bigquery/snippets/snippet.go

```diff
@@ -116,6 +116,39 @@ func listDatasets(client *bigquery.Client) error {
 	return nil
 }
 
+func printDatasetSimple(client *bigquery.Client, datasetID string) error {
```

Comment:
Sorry, maybe this should be something like printCoreDatasetInfo or printDatasetInfo, to avoid using "simple"?